examples/quickstart/disable_thread_stealing_executor.cpp

```diff
@@ -120,10 +120,10 @@ int main()
 
     // The following for_loop will be executed while thread stealing is disabled
     auto exec = executor_example::make_disable_thread_stealing_executor(
-        hpx::parallel::execution::par.executor());
+        hpx::execution::par.executor());
 
-    hpx::parallel::for_loop(hpx::parallel::execution::par.on(exec), 0, v.size(),
-        [](std::size_t i) {});
+    hpx::for_loop(
+        hpx::execution::par.on(exec), 0, v.size(), [](std::size_t i) {});
```

Comment:
Not to self: this change should be done in a separate PR for all occurrences.

Reply:
Yes, I only changed the ones close to the lines I changed. We'll eventually have to deprecate the names in `hpx::parallel::execution` which will help us change everything to the new names.